client/visualizer/src/static/css/style.css

```diff
@@ -86,11 +101,17 @@ form {
 
 #gamearea canvas {
     max-height: calc(100vh - 75px);
-    max-width: calc(100vw - 320px);
+    max-width: calc(100vw - 345px);
```

Comment:
.

Reply:
Sidebar has been made bigger to make space for the Profiler tab. As a consequence, the gamearea canvas has been made smaller so it doesn't flow underneath the sidebar.